lib/fake_stripe/stub_app.rb

```diff
@@ -5,8 +5,12 @@ class StubApp < Sinatra::Base
 
     # Charges
     post '/v1/charges' do
-      FakeStripe.charge_count += 1
-      json_response 201, fixture('create_charge')
+      if params['amount'] && params['amount'].to_i <= 0
```

Comment:
Is it valid to have a charge without an amount?

Reply:
With the actual Stripe API it is not valid. However, FakeStripe currently only tests all success cases, and you can see in all other specs literally no a parameters are being passed or checked. 

I felt this individually checking for this particular error scenario was the best compromise given the current behavior and how the test suite is currently written.